package.json

```diff
@@ -51,41 +51,41 @@
 	},
 	"devDependencies": {
 		"@sindresorhus/tsconfig": "^0.4.0",
-		"@types/chrome": "0.0.86",
+		"@types/chrome": "0.0.88",
 		"@types/codemirror": "0.0.76",
 		"@types/copy-webpack-plugin": "^5.0.0",
 		"@types/firefox-webext-browser": "^67.0.2",
 		"@types/jsdom": "^12.2.4",
-		"@types/mini-css-extract-plugin": "^0.2.1",
-		"@types/react": "^16.8.23",
+		"@types/mini-css-extract-plugin": "^0.8.0",
+		"@types/react": "^16.9.2",
 		"@types/terser-webpack-plugin": "^1.2.1",
-		"@typescript-eslint/eslint-plugin": "^1.11.0",
-		"@typescript-eslint/parser": "^1.11.0",
-		"ava": "^2.2.0",
+		"@typescript-eslint/eslint-plugin": "^1.13.0",
+		"@typescript-eslint/parser": "^1.13.0",
```

Comment:
I'm temporarily avoiding v2 because of a bug I don't know how to work around: https://github.com/typescript-eslint/typescript-eslint/issues/890

>   webpack.config.ts:undefined:undefined
>   ✖   0:0   Parsing error: If "parserOptions.project" has been set for @typescript-eslint/parser, ./webpack.config.ts must be included in at least one of the projects provided.         
> 
>   test/page-detect.ts:undefined:undefined
>   ✖   0:0   Parsing error: If "parserOptions.project" has been set for @typescript-eslint/parser, ./test/page-detect.ts must be included in at least one of the projects provided.       
> 
>   test/utils.ts:undefined:undefined
>   ✖   0:0   Parsing error: If "parserOptions.project" has been set for @typescript-eslint/parser, ./test/utils.ts must be included in at least one of the projects provided.             
> 
>   test/fixtures/globals.ts:undefined:undefined
>   ✖   0:0   Parsing error: If "parserOptions.project" has been set for @typescript-eslint/parser, ./test/fixtures/globals.ts must be included in at least one of the projects provided. 

They should go away by adding `test` in tsconfig's `includes`, but that file appears to be ignored.

Also reported in https://github.com/xojs/eslint-config-xo-typescript/issues/15